webserver/app/api.py

```
@@ -0,0 +1,31 @@
+app = FastAPI()
+
+@app.post("/add_task")
+async def add_task(request: AddTaskRequest):
+    if request.bucket_name != settings.allowed_bucket:
+        raise HTTPException(403, detail="Bucket not allowed")
+    
+    task = process_audio.delay(request.bucket_name, request.path)
+    return {"task_id": task.id}
+
+@app.get("/status/{task_id}")
+async def get_status(task_id: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_status`

(ANN201)